res/controllers/AKAI-MPKmini2-scripts.js

```diff
@@ -0,0 +1,139 @@
+
+var MPKmini = {};
+
+MPKmini.init = function() {
+    this.currentUnitNum = 1;
+    this.currentEffectNum = 1;
+    this.currentFlash = true;
+    this.timerId = 0;
+
+    MPKmini.updateLed();
+    this.timerId = engine.beginTimer(200, "MPKmini.flashLed()");
+};
+
+MPKmini.shutdown = function() {
+    if (this.timerId !== 0)
+        engine.stopTimer(this.timerId);
+    else
+        print("Shutdown: no timer to stop.");
+};
+
+MPKmini.getCurrentGroup = function(withEffect, withUnit) {
+    var grp = "";
+    if (withEffect) {
+        grp =  "[EffectRack1_EffectUnit" + MPKmini.currentUnitNum + "_Effect" + MPKmini.currentEffectNum + "]";
+    } else if (withUnit) {
+        grp =  "[EffectRack1_EffectUnit" + MPKmini.currentUnitNum + "]";
+    } else {
+        grp =  "[EffectRack1]";
+    }
+    print("get group: "+grp);
+    return grp;
+};
+
+MPKmini.resetLed = function() {
+    midi.sendShortMsg(0x90, 13, engine.getValue("[EffectRack1_EffectUnit1_Effect1]", "enabled") ? 127 : 0);
+    midi.sendShortMsg(0x90, 14, engine.getValue("[EffectRack1_EffectUnit1_Effect2]", "enabled") ? 127 : 0);
+    midi.sendShortMsg(0x90, 15, engine.getValue("[EffectRack1_EffectUnit1_Effect3]", "enabled") ? 127 : 0);
+    midi.sendShortMsg(0x90, 16, engine.getValue("[EffectRack1_EffectUnit1_Effect4]", "enabled") ? 127 : 0);
+    midi.sendShortMsg(0x90,  9, engine.getValue("[EffectRack1_EffectUnit2_Effect1]", "enabled") ? 127 : 0);
+    midi.sendShortMsg(0x90, 10, engine.getValue("[EffectRack1_EffectUnit2_Effect2]", "enabled") ? 127 : 0);
+    midi.sendShortMsg(0x90, 11, engine.getValue("[EffectRack1_EffectUnit2_Effect3]", "enabled") ? 127 : 0);
+    midi.sendShortMsg(0x90, 12, engine.getValue("[EffectRack1_EffectUnit2_Effect4]", "enabled") ? 127 : 0);
+};
+
+MPKmini.updateLed = function() {
+    MPKmini.resetLed();
+    MPKmini.flashLed();
+};
+
+MPKmini.flashLed = function() {
+    var start;
+    if (MPKmini.currentUnitNum === 1) {
+        start = 12;
+    } else {
+        start = 8;
+    }
+    midi.sendShortMsg(0x90, start + MPKmini.currentEffectNum, MPKmini.currentFlash ? 127 : 0);
+    MPKmini.currentFlash = !MPKmini.currentFlash;
+};
+
+/*
+ * Control mapping
+ */
```

Comment:
```suggestion
// Control mapping
```

We usually try to avoid block comments because you can't nest them, e.g. when commenting out large amounts of code when debugging.